src/test/java/org/saltyrtc/client/tests/signaling/peers/ResponderTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016-2018 Threema GmbH
+ *
+ * Licensed under the Apache License, Version 2.0, <see LICENSE-APACHE file>
+ * or the MIT license <see LICENSE-MIT file>, at your option. This file may not be
+ * copied, modified, or distributed except according to those terms.
+ */
+
+package org.saltyrtc.client.tests.signaling.peers;
+
+import org.junit.Test;
+import org.saltyrtc.client.signaling.peers.Responder;
+
+import static org.junit.Assert.assertEquals;
+
+public class ResponderTest {
+
+    @Test
+    public void testIdCounter() {
+        final Responder responder = new Responder((short) 1, 2);
```

Comment:
This test is confusing (since the ID should not be allowed at all). Maybe the id should also be validated in the `Peer` subclass constructors.
0x01 for the initiator, 0x02 <= id <= 0xff for the responder.

Reply:
Validating the id isn't a bad idea. I'll do that.

Reply:
Only downside is that we now validate twice, but I guess that won't hurt.